docs/source/installation/ubuntu.rst

```diff
@@ -70,6 +70,71 @@ Basic configuration options are provided in the default configuration file:
 Please note that by enabling hpfriends your conpot installation will automatically transmit attack data to The Honeynet
 Project. The fetch_public_ip option enables fetching the honeypot public ip address from a external resource.
 
+Init script
```

Comment:
Can you put this in a separate pull request? I highly recommend to use a branch per feature.